### PR TITLE
docs(intro): update wording, add note about next-nuxt, update seo title

### PR DESCRIPTION
**Pull request by MarkPieszak**
- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?
```
[x] Other - DOCS
```

image

### CODE REVIEW

**Comment by johnbiundo** [comment]
@MarkPieszak I like these changes.  This prompted me to start a chat with @kamilmysliwiec to discuss tightening up the usage of "Nest" vs. "NestJS" vs. "NestJs" in the docs.  I think it's good for "branding" to be consistent.  If he agrees, maybe we just switch over to using "NestJS" consistently everywhere.

I think @kamilmysliwiec  and I had a similar reaction regarding Nuxt/Next and thought that could be left out.  Do you feel strongly about that part?

Once I hear back from Kamil, we can move forward with the PR.  Thanks again for the contribution!

File: content/introduction.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Under the hood, Nest makes use of robust HTTP Server frameworks like [Express](h
 
 Nest provides a level of abstraction _above_ these common NodeJS frameworks (Express/Fastify), but also exposes their APIs directly to the developer. This allows developers the freedom to use the myriad of third-party modules which are available for the underlying platform.
 
-> Note: Remember Nest (or NestJS) is an extension/abstraction above NodeJS itself - although similary named to libraries like (Next.js/Nuxt.js) which are server-side rendering abstractions/wrappers around React & Vue respectively - **NestJS is intended to be a NodeJS API framework** and isn't similar or comparable to Nextjs & Nuxtjs.
```
Comment (BrunnerLivio) [inline content/introduction.md]:
```suggestion
> Note: Remember Nest (or NestJS) is an extension/abstraction above NodeJS itself - although similarly named to libraries like (Next.js/Nuxt.js) which are server-side rendering abstractions/wrappers around React & Vue respectively - **NestJS is intended to be a NodeJS API framework** and isn't similar or comparable to Next.js & Nuxt.js.
```

Some guys are really passionate about correct naming. Let's correctly adress them, so we do not piss anyone off 😹

**Comment by MarkPieszak** [comment]
Alright nuxt/next lines removed, added target="_blank" functionality to all external link functionality.

We should be good to go here :+1:

**Comment by johnbiundo** [comment]
👍 

@kamilmysliwiec Doc changes look great.  Are you good with the changes to the `markdown-compiler`?

**Comment by MarkPieszak** [comment]
I showed it to him this morning he was fine with it, long as it looks good to you!
Tested locally, worked great. :+1: @johnbiundo

**Comment by kamilmysliwiec** [comment]
I reverted this change:

> NestJS Introduction

Thanks for the PR!

**Comment by johnbiundo** [comment]
👍

**Pull request merged**